backend/User/views.py

```diff
@@ -79,7 +80,17 @@ def delete(self, request, user_id):
 			return Response(status=status.HTTP_404_NOT_FOUND)
 		except:
 			return Response(status=status.HTTP_400_BAD_REQUEST)
-		user.delete()
+		user.username = get_random_string(length=30)
```

Comment:
I don't know what was discussed yesterday but we might want do to something else than a random string here like "deleted user" or something, I would say (unless there's a good reason to randomise it)